.gitmodules

```diff
@@ -8,7 +8,9 @@
 	branch = origin/master
 [submodule "extern/PYB11Generator"]
 	path = extern/PYB11Generator
+	branch = update_for_cmake_exports
 	url = https://github.com/LLNL/PYB11Generator
 [submodule "extern/PolyClipper"]
 	path = extern/PolyClipper
+	branch = update_for_cmake_exports
 	url = https://github.com/LLNL/PolyClipper
```

Comment:
This was only put in place to ensure the CI worked in case it used this file. Once the main branches for the submodules were updated, I was going to remove these changes altogether.